docs/content/guides/columns/column-filter.md

```diff
@@ -155,765 +450,2860 @@ import 'handsontable/dist/handsontable.full.min.css';
+  return (
+    <HotTable
+      ref={hotTableComponentRef}
+      data={[
+        {
+          brand: 'Jetpulse',
+          model: 'Racing Socks',
+          price: 30,
+          sellDate: 'Oct 11, 2023',
+          sellTime: '01:23 AM',
+          inStock: false,
+        },
+        {
+          brand: 'Gigabox',
+          model: 'HL Mountain Frame',
+          price: 1890.9,
+          sellDate: 'May 3, 2023',
+          sellTime: '11:27 AM',
+          inStock: false,
+        },
+        {
+          brand: 'Camido',
+          model: 'Cycling Cap',
+          price: 130.1,
+          sellDate: 'Mar 27, 2023',
+          sellTime: '03:17 AM',
+          inStock: true,
+        },
+        {
+          brand: 'Chatterpoint',
+          model: 'Road Tire Tube',
+          price: 59,
+          sellDate: 'Aug 28, 2023',
+          sellTime: '08:01 AM',
+          inStock: true,
+        },
+        {
+          brand: 'Eidel',
+          model: 'HL Road Tire',
+          price: 279.99,
+          sellDate: 'Oct 2, 2023',
+          sellTime: '01:23 AM',
+          inStock: true,
+        },
+      ]}
+      columns={[
+        {
+          title: 'Brand',
+          type: 'text',
+          data: 'brand',
+        },
+        {
+          title: 'Model',
+          type: 'text',
+          data: 'model',
+        },
+        {
+          title: 'Price',
+          type: 'numeric',
+          data: 'price',
+          numericFormat: {
+            pattern: '$ 0,0.00',
+            culture: 'en-US',
+          },
+          className: 'htLeft',
+        },
+        {
+          title: 'Date',
+          type: 'date',
+          data: 'sellDate',
+          dateFormat: 'MMM D, YYYY',
+          correctFormat: true,
+          className: 'htRight',
+        },
+        {
+          title: 'Time',
+          type: 'time',
+          data: 'sellTime',
+          timeFormat: 'hh:mm A',
+          correctFormat: true,
+          className: 'htRight',
+        },
+        {
+          title: 'In stock',
+          type: 'checkbox',
+          data: 'inStock',
+          className: 'htCenter',
+        },
+      ]}
+      // enable the column menu
+      dropdownMenu={true}
+      // enable filtering
+      filters={true}
+      height="auto"
+      licenseKey="non-commercial-and-evaluation"
+    />
+  );
+};
 
-      item.className = 'item';
+/* start:skip-in-preview */
+ReactDOM.render(<App />, document.getElementById('exampleFilterOnInitialization'));
+/* end:skip-in-preview */
+```
 
-      const input = document.createElement('input');
+:::
 
-      input.type = 'checkbox';
-      input.name = 'cellData';
-      input.value = cellData;
-      input.checked = true;
+:::
 
-      input.addEventListener('change', event => this.onInputChange(event));
+## Filter as you type
 
-      const label = document.createElement('label');
+You can filter data by typing into a text box. This functionality is similar to dynamic filters in
```

Comment:
> The text input should find and filter out the values across all columns by default

@krzysztofspilka As far as I know, that's not possible. @adrianszymanski89 can you confirm?

Reply:
@kirszenbaum, we rechecked it and discussed it again with @AMBudnik, and it looks like, to the best of our knowledge, it will not be possible to do with the filters plugin. One solution would be to operate strictly on trimRows, entirely omitting the plugin, but I think it's not the point in this example.